lightgbm/src/main/scala/com/microsoft/azure/synapse/ml/lightgbm/params/LightGBMParams.scala

```diff
@@ -257,6 +257,14 @@ trait LightGBMPredictionParams extends Wrappable {
 
   def getFeaturesShapCol: String = $(featuresShapCol)
   def setFeaturesShapCol(value: String): this.type = set(featuresShapCol, value)
+
+  val predictDisableShapeCheck = new Param[Boolean](this, "predictDisableShapeCheck",
```

Comment:
instead of Param[Boolean] you need to use BooleanParam here

Reply:
@imatiach-msft 
I changed to `BooleanParam`. Please help re-run test again.

Thank you